start_cerberus.py

```diff
@@ -94,6 +99,22 @@ def main(cfg):
         while (int(iteration) < iterations):
             iteration += 1
 
+            # Check if api server url is ok
+            server_status = kubecli.is_url_available(api_server_url)
```

Comment:
I think checking the status of components needs to done after reporting in slack about the start of Cerberus monitoring to make sure user is notified about the start before performing any check when slack_integration is enabled.

Reply:
I have moved this check and the master node schedule check below the slack integration